words/words.md

```diff
@@ -636,11 +636,11 @@ Zum Glück reduziert der Sommer R, was die Situation verbessert:
 Oh.
 
 
-Überraschenderweise macht der Sommer die Spitzen schlimmer *und* regelmäßig! Das liegt daran, dass der Sommer die neuen <icon i></icon>s absenkt, was aber wiederum die neuen imunen <icon r></icon>s absenkt. Das bedeutet, Immunität stürzt im Sommer ab was eine große, wiederkehrende Spitze im Winter *erzeugt*.
+Überraschenderweise macht der Sommer die Spitzen schlimmer *und* regelmäßig! Das liegt daran, dass der Sommer die neuen <icon i></icon>s senkt, was aber wiederum die neuen immunen <icon r></icon>s reduziert. Das bedeutet, Immunität stürzt im Sommer ab, und *erzeugt* eine große, wiederkehrende Spitze im Winter.
```

Comment:
```suggestion
Überraschenderweise macht der Sommer die Spitzen schlimmer *und* regelmäßig! Das liegt daran, dass der Sommer die neuen <icon i></icon>s senkt, was aber wiederum die neuen immunen <icon r></icon>s senkt. Das bedeutet, Immunität stürzt im Sommer ab, und *erzeugt* eine große, wiederkehrende Spitze im Winter.
```
Ich finde es verständlicher wie im Original absichtlich zwei Mal das gleiche Wort zu verwenden. Es ist ja auch beide Male das gleiche gemeint. Sprachliche Varianz ist da für mich weniger wichtig.

Also entweder zwei Mal "senkt" oder zwei Mal "reduziert".

Reply:
Ich finde den Satz in beiden Varianten gleich verständlich und würde daher eher zur Varianten-Variante tendieren.